### ISSUE TITLE
Not working on US 915MHz - Can't complete JOIN

### ISSUE THREAD

**Issue by jonriber** [HelTecAutomation/ESP32_LoRaWAN#6]
Hello, anyone having this issue? I can't complete JOIN on TTN server. I'm using a single channel Lora gateway, just for technology validation, but i can't see any data on it.

**Comment by amirna2** [comment]
Yes, same issue. I downloaded the factory test that simply uses the 2 modules to send/receive lora packets and it won't join unless I use the 868E6 band.

**Comment by DeveloppeurPascal** [comment]
same here (France) on 868 band
